README.md

```diff
@@ -49,6 +49,7 @@
 - [Brackets](http://brackets.io/) - A modern, open source text editor that understands web design.
 - [Macvim](https://github.com/b4winckler/macvim) - Vim, the text editor.
 - [Sublime Text 3](http://www.sublimetext.com/) - The sophisticated text editor.
+- [Textmate](https://macromates.com/) - The missing editor for Mac OS X.
```

Comment:
I think it would be better to say `A graphical text editor.` referring to the [official repository](https://github.com/textmate/textmate)